src/csharp/Microsoft.Spark/Sql/Expressions/UserDefinedFunction.cs

```diff
@@ -34,10 +35,7 @@ internal sealed class UserDefinedFunction : IJvmObjectReferenceProvider
         {
             var pythonFunction = UdfUtils.CreatePythonFunction(jvm, command);
 
-            var dataType = (JvmObjectReference)jvm.CallStaticJavaMethod(
-                "org.apache.spark.sql.types.DataType",
-                "fromJson",
-                $"{returnType}");
+            var dataType = DataType.FromJson(jvm, returnType);
```

Comment:
it's not `var` anymore. Either `JvmObjectReference dataType` or just put it directly below. Can you also fix `var pythonFunction` right above?

Reply:
Sure, fixed both